src/lib/Components/Inbox/ActiveBids/ActiveBid.tsx

```diff
@@ -121,9 +121,12 @@ class ActiveBid extends React.Component<RelayProps, State> {
     const headline = `Lot ${lotNumber} · ${artistName} `
 
     const isInOpenLiveAuction = this.props.bid.sale && this.props.bid.sale.is_live_open
+    const bidderPositions = bid.sale_artwork.counts.bidder_positions
+    const bidderPositionsLabel = bidderPositions + " " + (bidderPositions === 1 ? "Bid" : "Bids")
+
     const subtitle = isInOpenLiveAuction
       ? "Live bidding now open"
-      : `Current Bid: ${bid.max_bid.display} \nHighest Bid: ${bid.sale_artwork.highest_bid.display}`
+      : `${bid.sale_artwork.highest_bid.display} (${bidderPositionsLabel})`
```

Comment:
Looks like all of this could maybe move into MP at some point (seems very shareable with other clients).